486_Project/Story_Class.py

```diff
@@ -0,0 +1,53 @@
+from Characters_Class import *
+class story:
+
+    def __init__(self):
+        pass
+
+    def select_class(self):
```

Comment:
I don't think it's a good idea to make a class method that depends on another class internally like this. It would be better to create a general method in your main code called select_class( Character_Class c), pass the Character object, and inside that method, set a Story attribute (i.e. player_class).

` class Story:
      player_class = <some default>
      def set_player_class( CharacterClass c): ...
   ----------- 
      // in main...
      def select_class():
                .....  // have your story text and input here
               story.set_player_class( Character_Class("warrior"))
      
`